packages/suite-native-atoms/src/Modal/BottomModal.tsx

```diff
@@ -17,10 +18,14 @@ type BottomModalProps = {
     onBackArrowClick?: () => void;
 };
 
-const modalWrapperStyle = prepareNativeStyle(utils => ({
+type WrapperStyleProps = {
+    insets: EdgeInsets;
```

Comment:
I would prefer to pass just `insetBottom: number` to make this little bit simplier

Reply:
[5ca9f07](https://github.com/trezor/trezor-suite/pull/5579/commits/5ca9f079c5a74efd47d083b6b62fc98ebe7fe64e)